media_driver/media_top_cmake.cmake

```diff
@@ -81,19 +81,34 @@ bs_ufo_link_libraries_noBsymbolic(
     "pciaccess m pthread dl rt"
 )
 
+find_package(PkgConfig)
+
 if (NOT DEFINED INCLUDED_LIBS OR "${INCLUDED_LIBS}" STREQUAL "")
-    # dep libs (gmmlib for now) can be passed through INCLUDED_LIBS, but if not, we need try to setup dep through including dep projects
-    if (NOT TARGET gmm_umd)
-        add_subdirectory("${BS_DIR_GMMLIB}" "${CMAKE_BINARY_DIR}/gmmlib")
+
+    # checking gmm dependencies
+    pkg_check_modules(LIBGMM igdgmm)
+
+    if(LIBGMM_FOUND)
+        include_directories(BEFORE ${LIBGMM_INCLUDE_DIRS})
+        link_directories(${LIBGMM_LIBRARY_DIRS})
```

Comment:
You can't call link_directories() after add_library(): it will not take effect. You can see this if you will configure and install gmmlib to the custom location. I get this error with your PR:
`/usr/bin/ld: cannot find -ligdgmm`

I have fixed this in my PR: https://github.com/intel/media-driver/pull/270. Please, take a look